ci/cloudbuild/builds/lib/bazel.sh

```diff
@@ -40,3 +40,17 @@ function bazel::common_args() {
   fi
   printf "%s\n" "${args[@]}"
 }
+
+# Bazel downloads all the dependencies of a project, as well as a number of
+# development tools during startup. In automated builds these downloads fail
+# from time to time due to transient network problems. Running `bazel fetch` at
+# the beginning of the build prevents such transient failures from flaking the
+# build.
+function bazel::fetch_dependencies() {
+  "./ci/retry-command.sh" 3 120 bazel fetch ... \
```

Comment:
(comment) I was hoping the network flakiness was mostly a kokoro issue and that we'd be able to avoid this w/ GCB. Oh well. (maybe I blame too much on kokoro ;-) )

Q: Why fetch and not sync? I think fetch requires us to specify specific targets, none of which make sense to me. Would just doing `bazel sync` be sufficient? (I don't know; asking)

Reply:
> (comment) I was hoping the network flakiness was mostly a kokoro issue and that we'd be able to avoid this w/ GCB. Oh well. (maybe I blame too much on kokoro ;-) )

I am just going to leave this here:

https://en.wikipedia.org/wiki/Fallacies_of_distributed_computing


> 
> Q: Why fetch and not sync? I think fetch requires us to specify specific targets, none of which make sense to me. Would just doing `bazel sync` be sufficient? (I don't know; asking)

As I recall `sync` does not download the indirect dependencies nor the SDKs (e.g. `@go_sdk//...`).  Candidly, I don't fully understand why the SDKs are needed, but they are.